Containerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:3.2.2
+
+ENV GEM_HOME="/usr/local/bundle"
+ENV PATH $GEM_HOME/bin:$GEM_HOME/gems/bin:$PATH
+
+# throw errors if Gemfile has been modified since Gemfile.lock
+RUN bundle config --global frozen 1
+
+WORKDIR /usr/src/app
+
+COPY Gemfile Gemfile.lock ./
+RUN bundle install
+
+COPY . .
```

Comment:
I did something similar too for wildfly.org except that I'm not copying the source but instead mount a volume when I run docker.

This allows to edit the doc and see the result live from Jekyll

Reply:
https://github.com/wildfly/wildfly.org/compare/main...jmesnil:wildfly.org:wildfly.org_in_docker